stories/assets/js/Tab_custom.js

```diff
@@ -1,16 +1,6 @@
 /* Tab JS start custom */
 export function TabFun() {
-  const $tab_item = $('.tab__item');
-  const $tab_content = $('.tab__content');
-  const $tab_item_active = $('.tab__item:first-child, .tab__content-wrapper .tab__content:first-child');
-
-  $tab_item_active.addClass('active');
-  $tab_item.click(function () {
-    const tab_id = $(this).attr('data-tab');
-    $tab_item.removeClass('active');
-    $tab_content.removeClass('active');
-    $(this).addClass('active');
-    $(`#${tab_id}`).addClass('active');
-  });
+   $(document).foundation();
+   $('.tabs li:first, .tabs-content .tabs-panel:first').addClass('is-active');
 }
 /* Tab JS end custom */
```

Comment:
@harishkumar-srijan remove this comment.

Reply:
done